runtime/hydradx/src/lib.rs

```diff
@@ -453,6 +454,10 @@ impl fp_rpc::ConvertTransaction<sp_runtime::OpaqueExtrinsic> for TransactionConv
 	}
 }
 
+pub fn is_testnet() -> bool {
+	Configuration::is_testnet()
+}
+
```

Comment:
this is little bit lost here and actually not that useful anyway. It is one-liner and only used once. 

I would suggest to remove it and use

`Configuration::is_testnet` directly in the TracksInfo::tracks.